docs/language-reference/inference.md

```diff
@@ -1,126 +1,147 @@
+--no-early-stop                   Disables early stopping in certain inference algorithms.
+--cps ARG                         Configuration of CPS transformation (only applicable to certain inference algorithms). The supported options are: none, partial, and full. Default: full.
+--dynamic-delay                   Runs dynamic delayed sampling on the model.
+--prune                           The model is pruned if possible.
+```
+
 ## Sequential Monte Carlo (SMC) Methods
 
 TreePPL supports two SMC methods:
 
   - bootstrap particle filter (BPF), and
   - alive particle filter (APF).
 
-Both inference methods have a resampling option.
-
 ### Bootstrap particle filter (BPF)
 
 ```bash
-tpplc -m smc-bpf
+tpplc <program> -m smc-bpf
 ```
 
 This is the classical SMC schema.
 Except in special cases, the alive particle filter (APF) should be preferred as the BPF can suffer from [path degeneration](../Explanation/path-degeneration.md).
 
 This inference is affected by the resample placement method.
 
-### Alive particle filter (APF)
+#### Options available
 
 ```bash
-tpplc -m smc-apf
+--particles INT                   The number of particles (i.e., samples or iterations). The default is 5000
+--subsample                       Whether to subsample the posterior distribution.
+--subsample-size INT              The number of subsamples to draw if --subsample is selected. Default: 1
+--resample ARG                    The selected resample placement method, for inference algorithms where applicable. The supported methods are: likelihood (resample immediately after all likelihood updates), align (resample after aligned likelihood updates, forces --align), and manual (sample only at manually defined resampling locations). Default: manual
+--resample-frac FLOAT             Floating point number to trigger resampling for SMC-BPF when ESS is less than resampleFrac × particleCount. Default: 0.7
+--cps ARG                         Configuration of CPS transformation (only applicable to certain inference algorithms). The supported options are: none, partial, and full. Default: full.
+--prune                           The model is pruned if possible.
+--dynamic-delay                   Runs dynamic delayed sampling on the model.
 ```
 
-This is the more advanced SMC schema.
-It avoids [path degeneration](../Explanation/path-degeneration.md) by keeping a population of particles alive.
-
-[Reference: Kudlicka et al. 2020](https://proceedings.mlr.press/v115/kudlicka20a.html).
-
-### Likelihood resampling
+### Alive particle filter (APF)
 
 ```bash
-tpplc -m smc-bpf --resample likelihood
+tpplc <program> -m smc-apf
 ```
 
-resamples the particles at every `observe`, `weight`, and `logWeight` statement.
-
-### Aligned resampling
-
-```bash
-tpplc -m smc-bpf --resample align
-```
+This is a more advanced SMC schema than BPF.
+It avoids [path degeneration](../Explanation/path-degeneration.md) by keeping a population of particles alive.
```

Comment:
I'm not certain this file is linked to or otherwise pointed at anywhere else, is it current and updated?